samples/switchless/README.md

```diff
@@ -118,10 +136,10 @@ The host first defines a structure specifically for configuring switchless calls
 first field `1` as the number of host worker threads for switchless OCALLs. In this example, 1) There is at most
 1 enclave thread all the time, and 2) The number of cores available to the host worker threads is unknown, and
 so we use 1 as explained above. The 2nd field specifies the number of enclave threads for switchless ECALLs.
```

Comment:
Where was it explained above?  70-73 only talk about using 1 when the number is 0 or negative, not when it's unknown.  I suspect you need to fix lines 70-73.

Reply:
I have completely rewritten the paragraph about configuring the enclave, providing the rationale/assumptions based on which the number of workers are chosen.